### PR TITLE
Gives the orphans a home

### PR DESCRIPTION
**Pull request by AffectedArc07**
**What does this PR do:**
Im talking about controllers, obviously. The votes and communications controllers have now been adopted into the hands of the StonedMC. This is part of my journey to unify everything into one controller, while making things make more sense. 

As for testing, votes still work, and so do radio channels, as well as atmospheric signalling. Everything **should** still work, but this is BYOND

**Changelog:**
:cl: AffectedArc07
add: SSvotes
add: SSradio
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Wow, two commits straight off. I'm closing this and banning you from git.

**Comment by AffectedArc07** [comment]
gosh darn

File: code/controllers/subsystem/vote.dm [hunk 1]
```diff
@@ -12,23 +13,10 @@ var/global/list/round_voters = list() //Keeps track of the individuals voting fo
...
 
-/datum/controller/vote/New()
-	if(vote != src)
-		if(istype(vote))
-			qdel(vote)
-		vote = src
-	spawn(0)
-		while(!QDELETED(src))
-			try
-				while(!QDELETED(src))
```
Comment (Fox-McCloud) [inline code/controllers/subsystem/vote.dm]: disgusting.

Comment (AffectedArc07) [reply code/controllers/subsystem/vote.dm]: image
In all seriousness what do you want me to change it do

**Comment by Fox-McCloud** [comment]
just need to update your CL.

**Comment by AffectedArc07** [comment]
@Fox-McCloud done

File: code/controllers/subsystem/radio.dm [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	"SyndTeam" 		= SYNDTEAM_FREQ,
+	"Supply" 		= SUP_FREQ,
+	"Service" 		= SRV_FREQ,
+	"AI Private"	= AI_FREQ,
+	"Medical(I)"	= MED_I_FREQ,
+	"Security(I)"	= SEC_I_FREQ
+	)
+	var/list/CENT_FREQS = list(ERT_FREQ, DTH_FREQ)
+	var/list/ANTAG_FREQS = list(SYND_FREQ, SYNDTEAM_FREQ)
+	var/list/DEPT_FREQS = list(AI_FREQ, COMM_FREQ, ENG_FREQ, MED_FREQ, SEC_FREQ, SCI_FREQ, SRV_FREQ, SUP_FREQ)
```
Comment (Fox-McCloud) [inline code/controllers/subsystem/radio.dm]: I admit, I don't like having lists that look like defines.

File: code/__DEFINES/radio.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#define RADIO_CHAT "radio_telecoms"
+#define RADIO_ATMOSIA "radio_atmos"
+#define RADIO_NAVBEACONS "radio_navbeacon"
+#define RADIO_AIRLOCK "radio_airlock"
+#define RADIO_SECBOT "radio_secbot"
+#define RADIO_HONKBOT "radio_honkbot"
+#define RADIO_MULEBOT "radio_mulebot"
+#define RADIO_CLEANBOT "10"
+#define RADIO_FLOORBOT "11"
+#define RADIO_MEDBOT "12"
```
Comment (Fox-McCloud) [inline code/__DEFINES/radio.dm]: *glaring intensifies*

What are you doing here 10, 11, and 12. What deep insights do you have to be a number as opposed to a string.

TELL ME YOUR SECRETS.

**Pull request merged**